client/src/BitsContainer.tsx

```diff
@@ -1,19 +1,26 @@
-import { Map } from 'immutable';
+import * as Immutable from 'immutable';
 import * as React from 'react';
 import { connect } from 'react-redux';
 import * as actionCreators from './action_creators';
 import Bit from './Bit';
 import { filterBits } from './bits_util';
+import { Bit as BitType } from './types';
 
-const BitsContainer = (props: any) => (
+type Props = {
+  bits?: Immutable.Map<string, BitType>
+  filters?: any
+  comments?: any
```

Comment:
Looks like we could just get rid of this

Reply:
mapStateToProps exports comments so it needs to be on this right?

Reply:
ah yes. this will come in the next pr. going to ignore this comment for now because then i won't have to merge lol.